packages/toolpad-components/src/DataGrid.tsx

```diff
@@ -285,11 +290,35 @@ const DataGridComponent = React.forwardRef(function DataGridComponent(
     selection,
     onSelectionChange,
     hideToolbar,
+    __toolpadComponents,
     ...props
   }: ToolpadDataGridProps,
   ref: React.ForwardedRef<HTMLDivElement>,
 ) {
   const nodeRuntime = useNode<ToolpadDataGridProps>();
+  const columnTypes = React.useMemo(
+    () => ({
+      ...CUSTOM_COLUMN_TYPES,
+      customComponent: {
+        renderCell: ({ value, colDef }: GridRenderCellParams) => {
+          const column = colDef as SerializableGridColumn;
+
+          const Component = __toolpadComponents[`codeComponent.${column.customComponent}`];
+
+          if (!Component) {
+            return (
+              <Typography variant="overline" sx={{ color: 'error.main', fontSize: '10px' }}>
+                No component selected
+              </Typography>
+            );
+          }
+
+          return <Component {...value} />;
```

Comment:
This doesn't seem right, do you really mean to spread the value as props?

Reply:
yes, because the value is what column has as a value in an array we pass to data grid